src/modules/Modal/ModalActions.js

```diff
@@ -30,11 +39,20 @@ ModalActions.propTypes = {
   /** An element type to render as (string or function). */
   as: customPropTypes.as,
 
+  /** Elements to render as Modal action buttons. */
+  actions: PropTypes.arrayOf(customPropTypes.itemShorthand),
+
   /** Primary content. */
   children: PropTypes.node,
 
   /** Additional classes. */
   className: PropTypes.string,
 }
 
+ModalActions.defaultProps = {
+  actions: [],
+}
```

Comment:
Since shorthand is disallowed with `children`, we'll want to avoid setting default values for it.  Otherwise, when a user uses `children` they will also have conflicting `actions` array defined unawares.

Instead, in the render method, just use `_.map` which will gracefully handle undefined values:

```diff
-{actions.map((action) => Button.create(action))}
+{_.map(actions, action => Button.create(action))}
```